astropy/units/imperial.py

```diff
@@ -67,6 +71,16 @@
          doc="U.S. customary tablespoon")
 def_unit(['tsp', 'teaspoon'], tbsp / 3, namespace=_ns,
          doc="U.S. customary teaspoon")
+def_unit(['Imp_gallon'], 4.54609 * si.liter, namespace=_ns,
```

Comment:
Do we need ``Imp_`` for these since they are already in the ``imperial`` namespace?

Reply:
We do since currently the `imperial` file defines a `gallon` by its US standard....  See https://github.com/astropy/astropy/pull/7090#issuecomment-358669909 and the subsequent discussion on names. The name `imperial.py` is rather unfortunate.